lib/daru/dataframe.rb

```diff
@@ -2249,18 +2274,6 @@ def split_by_category cat_name
       end
     end
 
-    # returns array of row tuples at given index(s)
-    def access_row_tuples_by_indexs *indexes
```

Comment:
@paisible-wanderer, please let us know the alternate method that can provide as the same functionality as `access_row_tuples_by_indexs `. 

```
irb(main):023:0> df
=> #<Daru::DataFrame(11x4)>
                  year   category   spending nb_spendin
          0       2010        dev         50          1
          1       2010        dev        150          1
          2       2010        dev        200          1
          3       2011        dev         50          1
          4       2012        dev        150          1
          5       2011     office        300          1
          6       2010     market         50          1
          7       2011     market        500          1
          8       2012     market        500          1
          9       2012     market        300          1
         10       2012        R&D         10          1
irb(main):024:0> df.access_row_tuples_by_indexs(1,2)
=> [[2010, "dev", 150, 1], [2010, "dev", 200, 1]]
irb(main):025:0> df
=> #<Daru::DataFrame(11x4)>
                  year   category   spending nb_spendin
          0       2010        dev         50          1
          1       2010        dev        150          1
          2       2010        dev        200          1
          3       2011        dev         50          1
          4       2012        dev        150          1
          5       2011     office        300          1
          6       2010     market         50          1
          7       2011     market        500          1
          8       2012     market        500          1
          9       2012     market        300          1
         10       2012        R&D         10          1
irb(main):026:0> df.access_row_tuples_by_indexs(1)
=> [[2010, "dev", 150, 1]]
irb(main):027:0> df.access_row_tuples_by_indexs(1,2)
=> [[2010, "dev", 150, 1], [2010, "dev", 200, 1]]
irb(main):028:0> df.access_row_tuples_by_indexs(1,2,3)
=> [[2010, "dev", 150, 1], [2010, "dev", 200, 1], [2011, "dev", 50, 1]]

```

I wondered, why there is no test case for this method.

Reply:
@paisible-wanderer
Somehow, I failed to realize that it was a public method...

re-adding something like
```def access_row_tuples_by_indexs *indexes
  get_sub_dataframe(indexes, by_position: false).each_row.map(&:to_a)
end```
to dataframe.rb should work....

I will have a look at [#463](https://github.com/SciRuby/daru/pull/463)

Reply:
@Shekharrajak
Somehow, I failed to realize that it was a public method...

re-adding something like
```
def access_row_tuples_by_indexs *indexes
  get_sub_dataframe(indexes, by_position: false).each_row.map(&:to_a)
end
```
to dataframe.rb should work....

I will have a look at [#463](https://github.com/SciRuby/daru/pull/463)